cmd/login/clientcert.go

```diff
@@ -467,9 +467,10 @@ func getAllOrganizationNamespaces(ctx context.Context, organizationService organ
 
 func fetchCluster(ctx context.Context, clusterService cluster.Interface, provider, namespace, name string) (*cluster.Cluster, error) {
 	o := cluster.GetOptions{
-		Namespace: namespace,
-		Name:      name,
-		Provider:  provider,
+		Namespace:      namespace,
+		Name:           name,
+		Provider:       provider,
+		FallbackToCapi: true,
```

Comment:
This flag indicates whether kgs should try to fetch the cluster as generic CAPI in case the cluster of the specified provider cannot be fetched. It prevents failures in commands, which depend on providers, in case the provider is unknown.